modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/IntrospectionUtils.java

```diff
@@ -211,8 +215,7 @@ else if (ArrayUtils.isEmpty(generics))
             }
             searchClass = searchClass.getSuperclass();
         }
-
-        throw new IllegalArgumentException(String.format("Class '%s' does not extend the '%s' class", type.getName(), superClass.getName()));
+        return new ArrayList<>();
```

Comment:
There is any particular reason to prefer a `LinkedList` instead of an `ArrayList`?